cluster/saltbase/salt/kube-controller-manager/default

```diff
@@ -17,6 +17,9 @@
 {% if pillar['node_instance_prefix'] is defined -%}
   {% set minion_regexp = "--minion_regexp='" + pillar['node_instance_prefix'] + ".*'" -%}
 {% endif -%}
```

Comment:
You may want be pedantic and
```
{% set cluster_name = "" -%}
```

here.

(I just tuned in for the Salt change to see if it was going to conflict badly in the same file.)